pkg/cortex/modules.go

```diff
@@ -170,6 +170,8 @@ func (t *Cortex) initRuntimeConfig() (services.Service, error) {
 
 func (t *Cortex) initOverrides() (serv services.Service, err error) {
 	t.Overrides, err = validation.NewOverrides(t.Cfg.LimitsConfig, tenantLimitsFromRuntimeConfig(t.RuntimeConfig))
+
+	t.API.RegisterOverrides(t.RuntimeConfig)
```

Comment:
This should be in `initRuntimeConfig` instead, after `t.RuntimeConfig` is set.

Reply:
Change done